benchmarking/scripts/metrics.js

```diff
@@ -11,14 +11,14 @@ async function main() {
   const blockNumber = blockHash.block.header.number;
 
   let totalExtrinsics = 0;
-
-  for (let i = blockNumber.toNumber() - 9; i <= blockNumber.toNumber(); i++) {
+  setTimeout(() => {}, 10000);
```

Comment:
explain the testing strategy in a human readable form, with a block of comments before the test that explains the global process